src/lib.rs

```diff
@@ -540,7 +540,7 @@ impl Library {
                     config.print_metadata(&meta);
                     self.framework_paths.push(PathBuf::from(val));
                 }
-                "-I" => {
+                "-I" | "-isystem" | "-iquote" | "-idirafter" => {
```

Comment:
`flag` is only 2 character flags here because of line 524

Reply:
Indeed, should also add a test for this :) Thanks!

I guess this means the parsing of the arguments has to be redone.

Reply:
This should be ok now, thanks for catching this! I've also added a test.